.github/workflows/azure-static-web-apps-gentle-beach-0eb2d061e.yml

```
@@ -7,6 +7,9 @@ on:
     paths:
       - "wordle-web/**"
   pull_request:
+    types: [opened, synchronize, reopened, closed]
+    branches:
+      - main

```

```suggestion
      - Assignment3
```
Should this be Assignment3? I know you guys are changingg main and then branching off of main but it might be easier to just change th workflow like you did in `Assignment2` brnach. This goes for the App service API yml as well

---

I personally like developing on main and using the assignment branches as "checkpoints" or turn-in versions of the project, but that would be more standard. We might switch back, we'll see - thanks!